app/models/view/Alert.java

```diff
@@ -113,7 +113,15 @@ public void setExtensions(final Map<String, Object> extensions) {
         _extensions = extensions;
     }
 
+    /**
+     * Returns the extensions.
+     *
+     * @return null if extensions are null, an unmodifiable map otherwise.
```

Comment:
Agreed. Further, I think you can leave this as an unprotected get (e.g. not wrapped in unmodifiable) as the setter above is not copy-write protected either. I'm okay with this since it's a view object and does not enter the codebase beyond the controller instance receiving/returning the view instance. @BrandonArp thoughts?